usmqe_tests/api/gluster/test_gluster_cluster.py

```diff
@@ -19,6 +20,111 @@
 ========
 """
 
+"""@pylatest api/gluster.cluster_create
+API-gluster: cluster_create
+***************************
+
+.. test_metadata:: author fbalak@redhat.com
+
+Description
+===========
+
+Positive create gluster cluster.
+"""
+
+
+def test_cluster_create_valid(
+        valid_session_credentials,
+        valid_nodes,
+        network_interface):
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+    """@pylatest api/gluster.cluster_import
+        .. test_step:: 1
+
+            Send POST request to Tendrl API ``APIURL/GlusterCreateCluster
+
+        .. test_result:: 1
+
+            Server should return response in JSON format:
+
+                {
+                  "job_id": job_id
+                }
+
+            Return code should be **202**
+                with data ``{"message": "Accepted"}``.
+
+        """
+    nodes = []
+    provisioner_ip = None
+    network = None
+    node_ids = []
+    for x in valid_nodes:
+        if "tendrl/server" in x["tags"]:
```

Comment:
I think it is better to get network from one of the storage nodes used for creating cluster because storage nodes can use different networks than server node.